flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/FlinkService.java

```diff
@@ -189,6 +286,12 @@ public boolean isJobManagerPortReady(Configuration config) {
         return savepointOpt;
     }
 
+    public void cancelSessionJob(JobID jobID, Configuration conf) throws Exception {
+        try (ClusterClient<String> clusterClient = getClusterClient(conf)) {
+            clusterClient.cancel(jobID).get(1, TimeUnit.MINUTES);
```

Comment:
We could use `operatorConfiguration.getFlinkClientTimeout()`.

Reply:
Fixed